packages/contracts-bedrock/README.md

```diff
@@ -267,13 +268,36 @@ Maintaining a consistent code style makes code easier to review and maintain, ul
 
 ## Deployment
 
-The smart contracts are deployed using `foundry` with a `hardhat-deploy` compatibility layer. When the contracts are deployed,
-they will write a temp file to disk that can then be formatted into a `hardhat-deploy` style artifact by calling another script.
+The smart contracts are deployed using `foundry`. The `DEPLOYMENT_OUTFILE` env var will determine the filepath that the
```

Comment:
We should link to https://github.com/ethereum-optimism/optimism?tab=readme-ov-file#development-and-release-process to make it clear that for production users should only be deploying from `op-contracts/vX.Y.Z` tags

Reply:
I can add some language about deploying releases if its a production deployment

Reply:
I have addressed this with a new section